engines/dradis-api/db/migrate/20150722125412_add_api_token_to_users.rb

```diff
@@ -0,0 +1,18 @@
+class AddAPITokenToUsers < ActiveRecord::Migration[5.1]
+  def up
+    add_column :users, :api_token, :string, null: false
+    add_index :users, :api_token, unique: true
+
+    User.transaction do
+      User.find_each do |user|
+        # Create an API token for all existing users
+        user.update_column(:api_token, SecureRandom.base64(18).tr("+/", "-_"))
```

Comment:
I know this is copied code, but in the spirit of leaving better than you found it:

1. This token is of a different length from the one defined in the file above
2. It doesn't use the `urlsafe` version
3. Applies a different set of 'readability' translations